environments/demo/demo.tfvars

```diff
@@ -49,6 +49,7 @@ frontends = [
       RCE = [
         "932100",
         "932110",
+        "932115",
```

Comment:
remove , from end

Reply:
the comma at the end seems to be the standard across all the environments